src/java.base/linux/classes/jdk/internal/platform/cgroupv2/CgroupV2Subsystem.java

```diff
@@ -359,7 +359,7 @@ private static Long ioStatLineToLong(String line, String[] matchNames) {
         if (line == null || EMPTY_STR.equals(line)) {
             return Long.valueOf(0);
         }
-        String[] tokens = line.split("\\s+");
+        String[] tokens = line.split(" ");
```

Comment:
This seems OK. According to https://docs.kernel.org/admin-guide/cgroup-v2.html#format